dataprofiler/profilers/datetime_column_profile.py

```diff
@@ -163,6 +170,15 @@ def _validate_datetime(date, date_format):
 
         return converted_date
 
+    @staticmethod
+    def _replace_day_suffix(date, suffix_list):
+        try:
+            for suffix in suffix_list:
+                date = date.replace(suffix, "")
+        except (AttributeError):
+            date = np.nan
```

Comment:
what error is occurring?

Reply:
When a non-string type is passed into this function and replace is called on the variable, it throws an AttributeError.